### PR TITLE
Feature/authorization

### PR DESCRIPTION
**Pull request by naturalnumber**
Merging work so far

### CODE REVIEW

File: backend_api/urls.py [hunk 1]
```diff
@@ -15,8 +15,12 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from rest_framework_simplejwt.views import (TokenObtainPairView, TokenRefreshView)
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-        path('revitalize', include('Revitalize.urls'))
+        path('admin/', admin.site.urls),
+        path('revitalize/', include('Revitalize.urls')),
```
Comment (lmsch) [inline backend_api/urls.py]: Can we change the endpoints from revitalize/ to api/

**Review by lmsch — CHANGES_REQUESTED** [review]

File: backend_api/urls.py (hunk 1 above)
Comment (naturalnumber) [reply backend_api/urls.py]: sure

Comment (naturalnumber) [reply backend_api/urls.py]: Can I do it for the next pull request, which will be immediately after this is accepted?
Unless the pull request includes commits after the request was made...

Comment (lmsch) [reply backend_api/urls.py]: It does yes.

**Review by lmsch — APPROVED** [review]

**Review by MatthewBowlan — APPROVED** [review]

**Pull request merged**